package.json

```diff
@@ -17,12 +17,20 @@
     "build:remix": "remix build",
     "dev": "run-p dev:*",
     "dev:css": "npm run generate:css -- --dev",
-    "dev:remix": "remix dev",
+    "dev:remix": "PORT=8080 remix dev",
```

Comment:
Adjust port to 8080 so its aligned with what we are running in e2e and avoid confusion

Reply:
`PORT` should always be set in the environment, not in a script/command.

Reply:
I will remove it, but is there a specific reason why we had to use port 8080 in e2e tests? We could use 3000 and avoid having the discrepancy running remix on a different port for diffferent scenarios.

Reply:
The tests are removed; that was a copy&paste error.